src/query/plan/planner.hpp

```diff
@@ -35,15 +35,20 @@ namespace plan {
 
 class PostProcessor final {
   Parameters parameters_;
+  std::vector<IndexHint> index_hints_{};
```

Comment:
put todo that if more hints are to be added, not related to indexes, that a super-struct needs to be added in order to account for all of the improvements in the future

Reply:
Added